Dockerfile

```diff
@@ -2,14 +2,14 @@ FROM zalando/openjdk:8u45-b14-6
 
 MAINTAINER Zalando SE
 
-COPY target/kio.jar /
-
 RUN mkdir /appdynamics
 COPY appdynamics /appdynamics
 
 EXPOSE 8080
 ENV HTTP_PORT=8080
 
-CMD java $(java-dynamic-memory-opts) $(newrelic-agent) $(appdynamics-agent) -jar /kio.jar
+CMD java $JAVA_OPTS $(java-dynamic-memory-opts) $(newrelic-agent) $(appdynamics-agent) -jar /kio.jar
+
+COPY target/kio.jar /
 
 ADD /scm-source.json /scm-source.json
```

Comment:
this should read target/scm-source.json generated by the lein-scm-source plugin (or am I wrong?)